Rubberduck.Core/UI/Controls/ToolBar.xaml

```diff
@@ -1,443 +1,444 @@
-                        </Trigger>
-                        <Trigger Property="HasDropShadow" SourceName="OverflowPopup" Value="true">
-                            <Setter Property="Margin" TargetName="Shdw" Value="0,0,5,5"/>
-                            <Setter Property="SnapsToDevicePixels" TargetName="Shdw" Value="true"/>
-                            <Setter Property="Color" TargetName="Shdw" Value="#71000000"/>
-                        </Trigger>
-                        <Trigger Property="Orientation" Value="Vertical">
-                            <Setter Property="Margin" TargetName="Grid" Value="1,3,1,1"/>
-                            <Setter Property="Style" TargetName="OverflowButton" Value="{StaticResource ToolBarVerticalOverflowButtonStyle}"/>
-                            <Setter Property="Height" TargetName="ToolBarThumb" Value="10"/>
-                            <Setter Property="Width" TargetName="ToolBarThumb" Value="Auto"/>
-                            <Setter Property="Margin" TargetName="ToolBarThumb" Value="-1,-3,0,0"/>
-                            <Setter Property="Padding" TargetName="ToolBarThumb" Value="5,6,6,1"/>
-                            <Setter Property="Margin" TargetName="ToolBarHeader" Value="0,0,0,4"/>
-                            <Setter Property="Margin" TargetName="PART_ToolBarPanel" Value="1,0,2,2"/>
-                            <Setter Property="DockPanel.Dock" TargetName="ToolBarThumb" Value="Top"/>
-                            <Setter Property="DockPanel.Dock" TargetName="ToolBarHeader" Value="Top"/>
-                            <Setter Property="HorizontalAlignment" TargetName="OverflowGrid" Value="Stretch"/>
-                            <Setter Property="VerticalAlignment" TargetName="OverflowGrid" Value="Bottom"/>
-                            <Setter Property="Placement" TargetName="OverflowPopup" Value="Right"/>
-                            <Setter Property="Margin" TargetName="MainPanelBorder" Value="0,0,0,11"/>
-                            <Setter Property="Background" Value="{StaticResource ToolBarVerticalBackground}"/>
-                        </Trigger>
-                        <Trigger Property="IsEnabled" Value="false">
-                            <Setter Property="Foreground" Value="{DynamicResource {x:Static SystemColors.GrayTextBrushKey}}"/>
-                        </Trigger>
-                    </ControlTemplate.Triggers>
-                </ControlTemplate>
-            </Setter.Value>
-        </Setter>
-        <Style.Triggers>
-            <DataTrigger Binding="{Binding Source={x:Static SystemParameters.HighContrast}}" Value="true">
-                <Setter Property="Background" Value="{DynamicResource {x:Static SystemColors.ControlBrushKey}}"/>
-            </DataTrigger>
-        </Style.Triggers>
-    </Style>
-
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"

```

Comment:
Where does this giant diff come from?

Reply:
It's actually two line changes. To see this, ignore the whitespace changes via the settings. I do not know why GItHub diff get confused on this.

Reply:
Can we please change the line endings back to line feeds only, as everywhere else in the repo?